src/css/components/_control.scss

```diff
@@ -1,4 +1,5 @@
-.video-js .vjs-control {
+.video-js .vjs-control,
+.video-js .vjs-button {
```

Comment:
I'll remove them, I initially added them in case we remove vjs-control from
the buttons in the future.
On Tue, Apr 28, 2015 at 6:07 PM Steve Heffernan <notifications@github.com>
wrote:

> In src/css/components/_control.scss
> <https://github.com/videojs/video.js/pull/2084#discussion_r29293744>:
>
> > @@ -1,4 +1,5 @@
> > -.video-js .vjs-control {
> > +.video-js .vjs-control,
> > +.video-js .vjs-button {
>
> Actually, we can remove all of these vjs-button additions in the CSS. All
> of the elements already have the vjs-control class name, so all of those
> styles will already apply.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/videojs/video.js/pull/2084/files#r29293744>.
>